### DISCUSSION TITLE
Question regarding the Version Support policy

### DISCUSSION THREAD

**Discussion by mooneeb** [signalfx/splunk-otel-collector#3632] [Q&A]
Hello 👋 👋 

Going through the documentation I wasn't able to find information regarding the version support policy for splunk-otel-collector. Could you please let me know what are the currently supported versions and what version support policy is followed? 

Thank you! 😃

**Comment by atoulme** [comment]
Please get in touch through support to discuss. Your account team and our product team will get in touch.
